F1WM/ApiModel/News/NewsSummary.cs

```diff
@@ -4,8 +4,7 @@ public class NewsSummary : NewsBase
 	{
 		public int CommentCount { get; set; }
 		public bool IsHighlighted { get; set; }
-		public string Type { get; set; }
-		public string TopicIcon { get; set; }
-		public int TopicId { get; set; }
+		public byte TypeId { get; set; }
+		public int TagId { get; set; }
```

Comment:
Calling it MainTagId would make it more clear that it's not the only tag associated with the news.